### PR TITLE
Remove border for screenshot

### LINKED ISSUE #133: Copied image has uneven black border
When the `Copy image` button is pressed, the resulting image saved to my clipboard has a black border around it, which is thicker on the left and top than the right and bottom. 

This is inconvenient, because when sharing screenshots I need to manually crop out this border first (because I never want a border). 

Here's an example: 
image

### PR DESCRIPTION
**Pull request by aparlato**
## Description

In mirror mode, a screenshot will temporarily remove the border to take the shot, then add it back.

This uses a small setTimeout function because otherwise it screenshots and leaves a white border (maybe after the border is removed but before another render resizes them?). I'm not in love with the flashing border you get, but I'm also not having much luck with alternative methods like CSS solutions having the border be its own div outside the map and position so that the map shows that border through while being on top.

I think we can either:
- go with this as-is and accept the flash
- get rid of the timeout/flash and accept a tiny white border
- make the flash include all the elements not included in the screenshot so it feels more intentional
- spend a little more time coming up with a CSS solution to let us ignore the whole border element without making the map smaller than it should be
- remove the border entirely at all times

I'm a little hesitant to always remove the border as things get a little hard to parse where one map ends and another begins if they're similar, but if cartographers prefer this or don't care, that's the easiest way forward.

Closes #133

## QA steps

- [ ] In mirror mode, copy screenshot of maps
- [ ] Paste somewhere (eg this PR) and see screenshot without borders
- [ ] Maperture maps continue to have borders after minor flash

### Create the PR

- [x] Fill out PR template
- [x] Make sure you've added a CHANGELOG entry under "Unreleased"
- [x] Request a review
- [ ] Make any requested changes and get approval

### After approval

- [ ] Merge any changes from `main` into your branch, resolve any conflicts
- [ ] Squash & merge changes into `main`
- [ ] Delete branch

### CODE REVIEW

**Comment by ebrelsford** [comment]
What do you think about making the `border-color` `transparent` only when we're taking a screenshot? That way we have no flicker and the transparent line is pretty easy to ignore?

image

Other than that maybe we could use `outline` since it's outside of the box rather than inside it?

**Comment by aparlato** [comment]
> What do you think about making the border-color transparent only when we're taking a screenshot? That way we have no flicker and the transparent line is pretty easy to ignore?

@ebrelsford yea, this does feel like the lesser evil here. I'm ok with that very small transparent border 👍 

> Other than that maybe we could use outline since it's outside of the box rather than inside it?

I just tried `outline` as well here, but couldn't get the desired effect because of how these maps are aligned with flexbox.

**Review by ebrelsford — APPROVED** [review]
Looks good!
